azure/pull-request.pipeline.yml

```diff
@@ -30,9 +30,9 @@ steps:
     CI=true yarn test
   displayName: 'Yarn run tests'
 
-# - script: |
-#    CI=true yarn test-cypress-with-new-test-api
-#   displayName: 'Yarn run cypress (e2e) tests'
+- script: |
```

Comment:
These will probably fail at the moment. Should we leave them in comments for now?